decider-HSMetrics/src/test/resources/expected_output/development/Tgl08studyTest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "workflowRunCount" : 0,
```

Comment:
Can this expected output be removed?  There is no "Tgl08studyTest" test case

Reply:
Fixed